City-of-Helsinki/helsinki-paatokset
# PP-0: Decision policymaker link and voting results refactoring

NOTE: This should be merged after this PR: https://github.com/City-of-Helsinki/helsinki-paatokset/pull/166

**To test**
- Checkout branch, run `make drush-cr drush-cim`
- Read the code changes. Does the refactored voting results code print out the same results as the previous version?
- If you don't have decision and case content, SSH into the container and run: `drush ap:fs;drush mim ahjo_decisionmakers:all;drush mim ahjo_cases:latest;drush mim ahjo_decisions:latest;drush mim ahjo_meetings:latest`
- Check that the other decisions link works on a case node: https://helsinki-paatokset.docker.so/fi/asia/hel-2019-013320
  - Navigate around using the dropdown and arrows. The link should still work (the URL will remain the same)
- Check that the voting results accordion works on these decisions:
  - https://helsinki-paatokset.docker.so/fi/asia/hel-2021-006118/0f854391-0585-c8cd-85df-7ed8d6400000
  - https://helsinki-paatokset.docker.so/fi/asia/hel-2016-012511/b4b8e27f-9763-4fd0-810c-96ada6c1c4b7
  - Navigate around using the dropdown. The voting results accordion should appear on the decisions where it is relevant
